__tests__/app.test.js

```diff
@@ -1,10 +1,27 @@
 const request = require('supertest');
-const app = require('../app');
+const app = require('../app.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/development-data/');
+
+afterAll (() => {
+    return db.end();
+})
+
+beforeEach(() => {
+    return seed(data);
+})
 
 describe('/api/topics', () => {
-    test('GET:200 sends an array of topic objects', () => {
+    test('GET:200 sends an array of topic objects with required properties', () => {
         return request(app)
             .get('/api/topics')
             .expect(200)
+            .then((response) => {
+                expect(response.body.topics).toEqual(expect.any(Array));
+                expect(Object.keys(response.body.topics[0])).toEqual(
```

Comment:
we should also test the values here, have a look at using a matcher method, ie. .toMatchObject or .objectContaining.